docs/voldoen-aan-wetten-en-regels/hulpmiddelen/standaarden.md

```diff
@@ -0,0 +1,144 @@
+
+### Praktische waarde van standaarden
+Geharmoniseerde normen hebben daarnaast een duidelijke praktische waarde. Ze vertalen abstracte wettelijke verplichtingen uit de AI-verordening naar concrete technische en organisatorische eisen, bijvoorbeeld op het gebied van risicomanagement en kwaliteitsmanagement, maar ook voor [duurzaamheid](../../onderwerpen/duurzaamheid.md) en [transparantie](../../onderwerpen/transparantie.md)​. Er zullen standaarden ontwikkeld worden voor de AI-verordening op het gebied van [verschillende fases in de algoritme-levenscyclus](../../levenscyclus/index.md). Zo helpen standaarden organisaties dus bij het ontwikkelen en implementeren van betrouwbare AI-systemen.
+
+## Een overzicht van belangrijke standaarden
+
+### Standaarden die worden ontwikkeld voor de AI-verordening
+Er is een standaardisatieverzoek gedaan door de Europese Commissie op 10 specifieke aspecten van AI. Deze aspecten vind je ook terug in de verschillende [onderwerpen](../../onderwerpen/index.md) die in het Algoritmekader terugkomen.
+
+1. Risicomanagement ("Risk Management")
+2. Gegevensbeheer en -kwaliteit ("Data Governance and Quality")
+3. Logging ("Record Keeping")
+4. Transparantie ("Transparency")
+5. Menselijke controle ("Human Oversight")
+6. Accuraatheid ("Accuracy")
+7. Robuustheid ("Robustness")
+8. Cybersecurity
+9. Kwaliteitsmanagement ("Quality Management")
+10. Conformiteitsassessment ("Conformity Assessment")
+
+Enkele van de belangrijkste standaarden die worden ontwikkeld voor het voldoen aan de vereisten voor hoog-risico AI systemen, zullen zijn:
+- AI trustworthiness framework
+- AI risk management
+- Quality management for EU AI regulatory purposes
+- Conformity assessment 
+
+#### Duurzaamheid
+Wat in deze 10 standaardisatieverzoeken nog niet terugkomt, maar wel in het Algoritmekader, is [duurzaamheid](../../onderwerpen/duurzaamheid.md). Momenteel worden er wel enkele TR's en TS's ontwikkeld. Zodra die bekend zijn, zullen deze worden aangevuld met meer informatie of links.
+
+- _[TR 20226](https://www.iso.org/standard/86177.html) 'Environmental sustainability aspects of AI systems'_ is een technisch rapport (TS), geen standaard. Dit TS bevindt zich in de 'committee draft' fase, wat wil zeggen dat er een concept is waar nog overeenstemming over moet worden gevonden.
+- _TR 18145 'Environmentally sustainable Artificial Intelligence'_ is een technisch rapport (TR). Het is nu "under approval".
+- Werkgroep JTC 21 heeft een "preliminary work item" genaamd _'Guidelines and metrics for the Environmental Impact of Artificial Intelligence Systems and Services'_ wat nog aan werk onderhevig is.
+
+### Welke standaarden, rapporten en specificaties zijn er al?
```

Comment:
Ik vraag me af of we deze lijst zo zonder toelichting moeten opnemen. Ik zou eigenlik liever een paar er uit lichten die voor alle organisaties die aan het begin staan relevant zijn. Dan denk ik bijvoorbeeld aan kwaliteit- en risicomanagement, het detecteren en mitigeren van bias, data governance.

Reply:
Ja het is inderdaad wel veel. Ik denk dat ik degenen op dat gebied laat staan en de rest weghaal dan. Weet jij nog een pagina van de NEN (of iets anders) waarnaar ik kan verwijzen voor het volledige overzicht, mochten mensen dat willen?